### ISSUE TITLE
Parse error with '#' in the input

### ISSUE THREAD

**Issue by dbohdan** [pstaender/csv2md#3]
By default csv2md fails to parse the following CSV file:
```
foo,bar
a#,"b"
```
printing the error
```
| foo | bar |
|---|---|
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: Invalid opening quote at line 2
    at Parser.__write (/home/dbohdan/.config/yarn/global/node_modules/csv-parse/lib/index.js:261:15)
    at Parser._transform (/home/dbohdan/.config/yarn/global/node_modules/csv-parse/lib/index.js:138:10)
    at Parser.Transform._read (_stream_transform.js:186:10)
    at Parser.Transform._write (_stream_transform.js:174:12)
    at doWrite (_stream_writable.js:397:12)
    at writeOrBuffer (_stream_writable.js:383:5)
    at Parser.Writable.write (_stream_writable.js:290:11)
    at ReadStream.ondata (_stream_readable.js:639:20)
    at emitOne (events.js:116:13)
    at ReadStream.emit (events.js:211:7)
```
Some thoughts on this:
* The cause seems to be with the comment parsing. The error goes away when you supply the command line argument `--csvComment ''`. If it is, this contradicts how the help message says that comments are detected at the beginning of the line.
* I haven't checked, but maybe the outdated version of `csv-parser` is to blame?
* CSV isn't exactly a standard, but [RFC-4180](https://tools.ietf.org/html/rfc4180) doesn't define comments and Microsoft Excel/LibreOffice Calc don't understand them. I'd disable them by default.

**Comment by pstaender** [comment]
fixed by #4

**Comment by dbohdan** [comment]
:+1:
